_tutorials/new-component/create-a-new-component-part-2.md

```diff
@@ -162,15 +163,19 @@ The first step is to install each component we want to use via [Bower](https://b
 
 With Bower configured, we can now run:
 <pre><code class="o-syntax-highlight--bash">bower install o-colors@^5.0.0 o-spacing@^2.0.0 o-typography@^6.0.0 o-buttons@^6.0.0 --save</code></pre>
+=======
```

Comment:
A merge conflict was commited. it looks like the section below the `===` is what we want

Reply:
Weird, I remember resolving that one -- must not have saved? good spot